mephisto/scripts/local_db/gh_actions/expire_all_units.py

```diff
@@ -0,0 +1,24 @@
+from mephisto.abstractions.databases.local_database import LocalMephistoDB
+
+"""
+This script should not be used locally.
+It is only used in a GitHub action as it does not automatically expire units.
```

Comment:
this script seems very "one-off"-y. is there no other case we may want to intentionally force expire units? I assume it could be useful? I would consider parametizing the task_name below and having this be a general script?

Reply:
I'm not aware of any situation outside of the GitHub action where you would want to run a script to expire units. Usually you would expire units by just doing ctrl-c two or three times on your mephisto task.

Reply:
"Force expiring" units is generally done with a `cleanup` script, which is provider-associated. This only updates the local status, and is a particular anti-pattern outside of the context of eval scripts.

Reply:
got it... if this script is really needed for a test, I guess I'm okay with keeping it then